### DISCUSSION TITLE
使用疑问：「 如何在minecraft应用 」

### DISCUSSION THREAD

**Discussion by CCCCCCTV** [Qv2ray/qv2ray.github.io#98] [Q&A]
比如说我想在hypixel服务器应用（hypixel.net     ip:104.17.71.15）
我该如何配置和使用qv2ray

**Comment by kslr** [comment]
https://qv2ray.net/manual/route.html#%E5%85%A8%E5%B1%80%E8%B7%AF%E7%94%B1%E8%A7%84%E5%88%99

**Comment by DuckSoft** [comment]
Minecraft 新版根本不走代理，你得用点别的黑科技劫持一下，比如 Windows/macOS 的 Proxifier 或者 Linux 的 proxychains 等等。
